roles/logstash/tasks/config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Create Logstash configuration files.
+  template:
+    src: "{{ item }}.j2"
```

Comment:
Nit: We don't really have .j2 on our templates. Also, we keep them in templates/ with similar path that they'll be deployed to.